emscripten-simulated-jerry-api/jerry.js

```diff
@@ -291,6 +298,20 @@
         return obj;
       }
       return weakModule(obj, gcCallback);
+    };
+    __jerry.strong = function(obj) {
+      // only worked for the weak ref
+
+      if (obj === null) {
+        return obj;
+      }
+      var typeStr = typeof obj;
+      // The `weak` module only works for objects, not for primitive values.
```

Comment:
Oh really?
The docs say:
> The Object can be a regular Object, an Array, a Function, a RegExp, or any of the primitive types or constructor function created with new.

"or any of the primitive types" suggests otherwise to me...?

https://www.npmjs.com/package/weak#weakref-weakobject-obj--function-callback

Reply:
yes, but I tried the weakModule with the primitive type, it can't work.